core/src/test/scala/org/broadinstitute/dsde/rawls/entities/local/CaseSensitivitySpec.scala

```diff
@@ -253,13 +259,16 @@ class CaseSensitivitySpec extends AnyFreeSpec with Matchers with TestDriverCompo
           }
         }
 
+        implicit val actorSystem = ActorSystem() // needed for stream materialization below
```

Comment:
Do you need to declare this twice? Seems you already have it at the top of the class (~Line 51-52ish)

Reply:
good catch; removed.